terasoluna-gfw-common-libraries/terasoluna-gfw-common/src/main/java/org/terasoluna/gfw/common/codelist/i18n/SimpleI18nCodeList.java

```diff
@@ -323,7 +323,8 @@ public void setColumns(Map<String, Map<Locale, String>> cols) {
 
     /**
      * Sets the default locale as fallback.<br>
-     * Defaults to {@link Locale#getDefault()}
+     * If extend this and override default value of {@link SimpleI18nCodeList#fallbackTo fallbackTo}, affects
+     * {@link SimpleI18nCodeList#afterPropertiesSet afterPropertiesSet}.
```

Comment:
- [ ] I think you should write warning on Javadoc of `fallbackTo`. `setFallbackTo` is not related directly.